modules/gapi/src/executor/gexecutor.hpp

```diff
@@ -47,50 +44,31 @@ namespace gimpl {
 //
 // Today's exchange data objects are:
 // - cv::Mat               - for image buffers
+// - cv::MediaFrame        - for media buffers
 // - cv::Scalar            - for single values (with up to four components inside)
 // - cv::detail::VectorRef - an untyped wrapper over std::vector<T>
+// - cv::detail::OpaqueRef - an untyped wrapper over an object with an arbitrary type T
```

Comment:
Is it a valid change from me? I'm just not sure these types used for exchanging between Islands

Reply:
Looks valid